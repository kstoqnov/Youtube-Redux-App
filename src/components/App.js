import React from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';
import HomePage from './views/HomePage';
import VideoDetailPage from './views/VideoDetailPage';

export default () => {
  return (
    <Router>
      <div>
        <Route exact path="/" component={HomePage} />
        <Route path="/video-detail:id" component={VideoDetailPage} />

      </div>
    </Router>
  );
};
