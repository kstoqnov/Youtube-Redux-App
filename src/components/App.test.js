import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

describe('App', () => {
  it('Should snapshot App', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
