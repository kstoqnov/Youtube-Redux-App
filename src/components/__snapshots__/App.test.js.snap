
exports[`App Should snapshot App 1`] = `
<BrowserRouter>
  <div>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/video-detail:id"
    />
  </div>
</BrowserRouter>
`;
